.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
```

Comment:
NICE! 

TIP: i love adding emoticons in the title. It provides a very nice visual on the commit changes e.g. when you look at the last commit in the code tree. Checkout https://gitmoji.dev/ to get fast an emoticon

I downloaded them all with a script


🔨 Add or update development scripts.
- 🩹 Simple fix for a non-critical issue.
- ⚗️ Perform experiments.
- 👽️ Update code due to external API changes.
- 🚑️ Critical hotfix.
- 💫 Add or update animations and transitions.
- ⬇️ Downgrade dependencies.
- ⬆️ Upgrade dependencies.
- 🎨 Improve structure / format of the code.
- 🍻 Write code drunkenly.
- 🍱 Add or update assets.
- 🔖 Release / Version tags.
- 💥 Introduce breaking changes.
- 🧱 Infrastructure related changes.
- 🐛 Fix a bug.
- 🏗️ Make architectural changes.
- 💡 Add or update comments in source code.
- 👥 Add or update contributor(s).
- 📸 Add or update snapshots.
- 🗃️ Perform database related changes.
- 📈 Add or update analytics or track code.
- 🚸 Improve user experience / usability.
- 🔐 Add or update secrets.
- 🤡 Mock things.
- ⚰️ Remove dead code.
- 🚧 Work in progress.
- 👷 Add or update CI build system.
- 🥚 Add or update an easter egg.
- 🔥 Remove code or files.
- 🌐 Internationalization and localization.
- 🥅 Catch errors.
- 💚 Fix CI Build.
- 🔨 Add or update development scripts.
- ➖ Remove a dependency.
- ➕ Add a dependency.
- 📱 Work on responsive design.
- 🏷️ Add or update types.
- 💄 Add or update the UI and style files.
- 🔒️ Fix security issues.
- 🔊 Add or update logs.
- 🔍️ Improve SEO.
- 📝 Add or update documentation.
- 🧐 Data exploration/inspection.
- 🔇 Remove logs.
- 👔 Add or update business logic
- 📦️ Add or update compiled files or packages.
- 📄 Add or update license.
- 🛂 Work on code related to authorization, roles and permissions.
- ✏️ Fix typos.
- 💩 Write bad code that needs to be improved.
- 📌 Pin dependencies to specific versions.
- ♻️ Refactor code.
- ⏪️ Revert changes.
- 🚀 Deploy stuff.
- 🚨 Fix compiler / linter warnings.
- 🙈 Add or update a .gitignore file.
- 🌱 Add or update seed files.
- ✨ Introduce new features.
- 💬 Add or update text and literals.
- 🩺 Add or update healthcheck.
- 🎉 Begin a project.
- 🧑‍💻 Improve developer experience
- 🧪 Add a failing test.
- 🚩 Add, update, or remove feature flags.
- 🚚 Move or rename resources (e.g.: files, paths, routes).
- 🔀 Merge branches.
- 🗑️ Deprecate code that needs to be cleaned up.
- ♿️ Improve accessibility.
- ✅ Add, update, or pass tests.
- 🔧 Add or update configuration files.
- ⚡️ Improve performance.